docs/content/dagster-cloud/developing-testing/code-locations.mdx

```diff
@@ -1,7 +1,245 @@
+
+### Deleting code locations
+
+To delete a code location, click the **dropdown menu** to the right of the location. In the menu, click **Remove**:
+
+<Image
+alt="Highlighted Remove option in the dropdown menu next to a code location in Dagster Cloud"
+src="/images/dagster-cloud/developing-testing/code-locations/delete-code-location.png"
+width={1940}
+height={480}
+/>
+
+When prompted, confirm the deletion.
+
+---
+
+## Managing code locations using the dagster-cloud CLI
+
+You can also use the `dagster-cloud workspace` CLI commands to:
+
+- [Add and update code locations](#adding-and-updating-code-locations)
+- [Delete code locations](#deleting-code-locations-1)
+- [Sync the workspace and remote](#syncing-the-workspace)
+
+These commands perform the same underlying operations as editing your code locations in the **Workspace** tab in Dagster Cloud. Refer to the [dagster-cloud CLI guide](/dagster-cloud/developing-testing/dagster-cloud-cli) for more info and installation instructions.
+
+### Adding and updating code locations
+
+You can add or update a code location with the `add-location` command. For example, to add our public example image, you can run:
+
+```shell
+# Set up YAML file for example location
+cat > example_location.yaml <<EOL
+location_name: cloud-examples
+image: dagster/dagster-cloud-examples:latest
+code_source:
+  package_name: dagster_cloud_examples
+EOL
+
+dagster-cloud workspace add-location --from example_location.yaml
+```
+
+Instead of creating a YAML file, you may also pass these values inline as command line options:
+
+```shell
+dagster-cloud workspace add-location test_location \
+    --image dagster/dagster-cloud-examples:latest \
+    --package-name dagster_cloud_examples
+```
+
+You may also selectively overwrite parts of your YAML input using inline options:
+
+    dagster-cloud workspace add-location \
+        --from example_location.yaml \
+        --image dagster/dagster-cloud-examples:1d9c5d
+
+The arguments to the `add-location` and `update-location` commands are similar to the [keys in the YAML config editor in the **Workspace** tab](#adding-code-locations). Run `dagster-cloud workspace add-location --help` to see all available options.
+
+**Note**: If updating an existing code location, the full set of information about the location must be specified even if only one piece of configuration is modified.
+
+### Deleting code locations
+
+Delete an existing code location from Dagster Cloud using the `delete-location` command:
+
+```shell
+dagster-cloud workspace delete-location test_location
+```
+
+### Syncing the workspace
+
+You can also keep the YAML configuration for your entire workspace in a file and use the `dagster-cloud sync` command to reconcile the workspace config in Dagster Cloud with that local file.
+
+For example, if you have the following `cloud_workspace.yaml` file:
+
+```yaml caption=cloud_workspace.yaml
```

Comment:
just noting that it's a very common point of confusion that this has a different format than the open_source workspace.yaml while having many of the same keys, we could address that here

Reply:
Noted. I'm going to merge this and make a note to come back to it before Tuesday.